nested/premium-files-nfs.json

```diff
@@ -0,0 +1,103 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "lampCommon": {
+            "metadata": {
+                "description": "Common LAMP values"
+            },
+            "type": "object"
+        }
+    },
+    "resources": [
+        {
+            "type": "Microsoft.Storage/storageAccounts",
+            "apiVersion": "2019-06-01",
+            "location": "[parameters('lampCommon').location]",
+            "name": "[variables('storageName')]",
+            "kind": "FileStorage",
+            "sku": {
+                "name": "[parameters('lampCommon').storageAccountType]",
+                "tier": "Premium"
+            },
+            "properties": {
+                "publicNetworkAccess": "Enabled",
+                "largeFileSharesState": "Enabled",
+                "minimumTlsVersion": "TLS1_2",
+                "networkAcls": {
+                    "bypass": "AzureServices",

```

Comment:
what does this do?

Reply:
These settings translate to allowing storage access to only designated virtual network. i.e VMSS subnet in our case.